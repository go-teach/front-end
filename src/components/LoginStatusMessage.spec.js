import React from 'react';
import { shallow } from 'enzyme';
import LoginStatusMessage from './LoginStatusMessage';

describe('LoginStatusMessage component', () => {
	it('should return proper component', () => {
		const wrapper = shallow(<LoginStatusMessage isLoggedIn={true}/>);
		expect(wrapper).toMatchSnapshot();
	});
});
