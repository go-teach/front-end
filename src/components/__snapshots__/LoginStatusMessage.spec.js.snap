// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginStatusMessage component should return proper component 1`] = `
<Component>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }
  >
    You are "logged in" right now
  </Text>
</Component>
`;
